toolkit/docs/how_it_works/0_intro.md

```diff
@@ -2,23 +2,308 @@ How The Build System Works
 ===
 ## Next: [Initial Prep](1_initial_prep.md)
 ## General Build Flow
-Graph showing how the various build components interact.
 
-Edges are marked with the tool used in blue when applicable.
+This section is intended to give an overview of Mariner's build process and toolkit. The diagrams below follow conventions stated in the key.
+```mermaid
+---
+title: Flowchart key
+---
+flowchart TD
+    %% style definitions
+    classDef io fill:#247BA0,stroke:#333,stroke-width:2px,color:#fff;
+    classDef process fill:#B05E2F,stroke:#333,stroke-width:2px,color:#fff;
+    classDef decision fill:#51344D,stroke:#333,stroke-width:2px,color:#fff;
+    classDef goodState fill:#566E40,stroke:#333,stroke-width:2px,color:#fff;
+    classDef badState fill:#BC4B51,stroke:#333,stroke-width:2px,color:#fff;
 
-![Build Flow Graph](images/flow.png)
+    %% node definitions
+    input[/input or output/]:::io
+    process[process]:::process
+    decision{{decision}}:::decision
+    goodstate([good state]):::goodState
+    badstate[[bad state]]:::badState
```

Comment:
Let's use the same symbol (different colours are OK) for the good and bad state. These are both terminal nodes, while what we draw for the bad state now is [commonly interpreted as a pre-defined process](https://en.wikipedia.org/wiki/Flowchart).

```suggestion
    goodstate([good state]):::goodState
    badstate([bad state]):::badState
```

Reply:
I originally had this; however, this is not be sufficient for Green/red colorblindess. Could lead to confusion if error is the inteded outcome or not.

Reply:
In that case let's use [blue and red](https://www.datylon.com/blog/data-visualization-for-colorblind-readers#:~:text=The%20first%20rule%20of%20making,out%20of%20these%20two%20hues.) for instance. What do you think?

Reply:
Ok, so in an effort to keep `states` the same shape, I instead changed the key to reflect that all error states = bad states so that the label `Error` can also be used as an idicator :)